packages/core/types/src/types/core/config/admin.ts

```diff
@@ -0,0 +1,49 @@
+export interface ApiTokenProp {
+  salt: string;
+}
+
+export interface AuthProp {
+  secret: string;
+}
+
+export interface TransferTokenProp {
+  salt: string;
+}
+
+export interface AuditLogsProp {
+  retentionDays?: number;
+}
+
+export interface ForgotPasswordProp {
+  emailTemplate?: string;
```

Comment:
If you're passing the forgotPassword prop, shouldn't all these fields be mandatory?